Doc/using/configure.rst

```diff
@@ -185,6 +185,13 @@ General Options
 
    .. versionadded:: 3.11
 
+.. cmdoption:: --disable-gil
+
+   Enables experimental support for running Python without the global
+   interpreter lock (GIL).
```

Comment:
You can add a link to the glossary:

```suggestion
   Enables experimental support for running Python without the
   :term:`global interpreter lock` (GIL).
```

see: https://docs.python.org/dev/glossary.html#term-global-interpreter-lock